packages/noports_core/lib/src/sshrv/sshrv_impl.dart

```diff
@@ -34,10 +58,36 @@ class SshrvImplExec implements Sshrv<Process> {
         'N.B. sshnp is expected to be compiled and run from source, not via the dart command.',
       );
     }
+    var rvArgs = [
+      '-h',
+      host,
+      '-p',
+      streamingPort.toString(),
+      '--local-port',
+      localPort.toString(),
+    ];
+    if (bindLocalPort) {
+      rvArgs.add('--bind-local-port');
+    }
+    Map<String, String> environment = {};
+    if (rvdAuthString != null) {
+      rvArgs.addAll(['--rv-auth']);
+      environment['RV_AUTH'] = rvdAuthString!;
```

Comment:
Adding to environment rather than command line to prevent any potentially interesting info from being unnecessarily visible

Reply:
We should have good documentation explaining how the flag + environment variable works in tandem

Reply:
I think including this in the usage is good enough, but it needs to be well written.